articles/data-factory/concepts-pipeline-execution-triggers.md

```diff
@@ -322,6 +322,9 @@ Tumbling window triggers are a type of trigger that fires at a periodic time int
 
 For more information about tumbling window triggers and, for examples, see [Create a tumbling window trigger](how-to-create-tumbling-window-trigger.md).
 
+> [!IMPORTANT]
```

Comment:
Instead of !Important, lets do !Note. Changed the order of text  a bit 

> [!NOTE]
> The status of the tumbling window trigger reflects the state of the triggered pipeline run. This is different than the 'fire and forget' behavior of the schedule trigger which is successful as long as a pipeline run started.